src/libraries/System.Private.CoreLib/src/System/IO/FileStatus.Unix.cs

```diff
@@ -182,7 +183,7 @@ internal FileAttributes GetAttributes(ReadOnlySpan<char> path, ReadOnlySpan<char
         {
             EnsureCachesInitialized(path, continueOnError);
 
-            if (!_exists)
+            if (!EntryExists)
```

Comment:
Usage of braces

Reply:
I kept the braces as they were because this style is used throughout the file. I didn't want to go reformat only part of it, or start touching unrelated parts.

Reply:
Ah okay. Thank you :smile: